src/gui/sdlmain.cpp

```diff
@@ -3691,19 +3691,6 @@ bool GFX_Events()
 					sdl.draw.callback(GFX_CallBackRedraw);
 				GFX_UpdateMouseState();
 
-				// When we're fullscreen, the DOS program might
-				// change underlying draw resolutions, which can
-				// pose a problem when switching back to
-				// window-mode (beacuse the prior window-size
-				// may not longer accomodate the new draw-size).
-				// So in this case, we want to reset the screen
-				// size - but only if we're not in the middle of
-				// resizing the window (which also fires EXPOSED
-				// events).
-				if (!sdl.desktop.fullscreen &&
```

Comment:
What's strange about this comment is this part: "When we're fullscreen, the DOS program might change underlying draw resolutions". Well, this `GFX_ResetScreen` will only happen if we're _not_ in fullscreen.

Reply:
The scenario involved starting fullscreen in a graphical mode (say, 320x200), then while still fullscreen, exiting back out to a text mode (larger resolution), and then toggling out of fullscreen.

@bmunger reported it, exiting out of (either Quake or Blood?) - where the text screen was cutoff a bit.
It was easily reprodiceable at the time, but now I can reproduce it (and tried many resolutions and output modes).  

If it can go away, then all the better! The fewer corner cases in SDL main the better!

@bmunger - maybe you can remember the scenario better?